flink-streaming-java/pom.xml

```diff
@@ -41,21 +42,21 @@ under the License.
 		<dependency>
 			<groupId>org.apache.flink</groupId>
 			<artifactId>flink-core</artifactId>
-			<version>${project.version}</version>
+			<version>${project.parent.version}</version>
```

Comment:
these pom changes are unnecessary

Reply:
Removing them. (I am not 100% sure on the release process of these artifacts). Will sync offline to understand more about it.